packages/website/src/features/Deploy/QueueFromGitOpsPage.tsx

```diff
@@ -391,6 +391,7 @@ function QueueFromGitOps() {
       try {
         await switchChainAsync({ chainId: currentSafe?.chainId || 1 });
         buildInfo.doBuild();
+        return;
```

Comment:
This is a bug fix discovered during the development of this PR, in order to prevent executing `doBuild()` twice, since it is call also outside this if statement